examples/tutorials/CFD_Theory_Tutorial.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+  "nbformat": 4,
```

Comment:
Check the https://lorenabarba.com/blog/cfd-python-12-steps-to-navier-stokes/ tutorial series for code and examples